boards/wch/ch32v003evt/doc/index.rst

```diff
@@ -0,0 +1,116 @@
+.. _wch_ch32v003evt:
+
+WCH CH32V003EVT
+###############
+
+Overview
+********
+
+The `WCH`_ CH32V003EVT hardware provides support for QingKe 32-bit RISC-V2A
+processor and the following devices:
+
+* CLOCK
+* :abbr:`GPIO (General Purpose Input Output)`
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+
+.. figure:: img/wch_ch32v003evt_front.webp
+     :align: center
+     :alt: WCH CH32V003EVT
+
+     WCH CH32V003EVT (Credit: WCH)
```

Comment:
```suggestion
```

you may actually drop this as the illustration will be auto-added by the `zephyr:board::` directive